src/components/page/page.stories.tsx

```diff
@@ -18,7 +18,7 @@ import { EuiPageSection } from './page_section';
 import { EuiPage, EuiPageProps } from './page';
 
 const meta: Meta<EuiPageProps> = {
-  title: 'EuiPage',
+  title: 'Templates/EuiPage',
```

Comment:
[EuiPage](https://eui.elastic.co/#/layout/page-components) itself is under the Layout section in our docs, rather than templates - I believe `EuiPageTemplate` is the only template
```suggestion
  title: 'Layout/EuiPage',
```

Reply:
Yes, that's my bad - I put them all together 🙈 Thanks for the catch!